### PR TITLE
fix (ui/vue): prevent infinite looping steps

### PR DESCRIPTION
**Pull request by nicoalbanese**
This pr fixes issue where useChat with vue automatically sends tool results back to the model

This would have been previously correct with maxToolRoundtrips.

### CODE REVIEW

**Review by lgrammel — APPROVED** [review]

**Review by lgrammel — CHANGES_REQUESTED** [review]
Requires changeset

**Comment by lgrammel** [comment]
You can add a changeset by running `pnpm changeset` in the project root. Please make it a patch for the vue provider.

**Comment by nicoalbanese** [comment]
> You can add a changeset by running `pnpm changeset` in the project root. Please make it a patch for the vue provider.
> Requires changeset

Added! Let me know if this works.

File: .changeset/young-clocks-end.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@ai-sdk/vue': patch
+---
+
+fix maxSteps
```
Comment (lgrammel) [inline .changeset/young-clocks-end.md:5]: please change to `fix (ui/vue): prevent infinite looping steps`

**Review by lgrammel — APPROVED** [review]

**Pull request merged**